concepts/array-analysis/introduction.md

```diff
@@ -1,6 +1,7 @@
 # Introduction
```

Comment:
General comment: You can add yourself as author for the concept and as contributor for the exercise (see the respective config.json).